aiida/schedulers/plugins/test_slurm.py

```diff
@@ -46,43 +54,46 @@ def test_parse_common_joblist_output(self):
 
         job_list = scheduler._parse_joblist_output(retval, stdout, stderr)
 
+        def get_job(id):
```

Comment:
`id` is a reserved keyword but I guess this file is in pre-commit exclude. Maybe `job_id`is better